common/schemes/wc_stasis_trap_scheme.txt

```diff
@@ -0,0 +1,175 @@
+﻿stasis_trap_scheme = {
```

Comment:
It says I'm discovered though I'm not.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/117575175-b712c680-b0f1-11eb-8031-4d351a18f389.png)

</details>